frontend/src/tests/offer-tests.js

```diff
@@ -74,9 +74,30 @@ export function offersTests(api) {
         expect(newOffer.emailed_date).not.toBeNull();
     });
 
-    it.todo(
-        "when an offer is emailed a second time, the emailed date is updated to the most recent emailed date"
-    );
+    it("when an offer is emailed a second time, the emailed date is updated to the most recent emailed date", async () => {
+        // record previous emailed_date
+        const resp = await apiGET(
+            `/admin/assignments/${assignment.id}/active_offer`
+        );
+        expect(resp).toHaveStatus("success");
+        let previousDate = resp.payload.emailed_date;
```

Comment:
Can we get an active offer from the database seeder rather than using a fetch to get the offer? It's important if we can because using the database seeder would allow us to avoid any issues that might arise from a failed fetch.